src/ProgressOnderwijsUtils/Win32/VirusScan.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+using Microsoft.Windows.Sdk;
+using ProgressOnderwijsUtils;
+
+namespace ProgressOnderwijsUtils.Win32
+{
+    public static class VirusScan
+    {
+        public static bool IsMalware(string content, string contentName, string sessionName)

```

Comment:
We gebruiken vermoedelijk maar 1 overload echt, dus dan kan die andere (vermoedelijk deze) wel weg lijkt me.

Reply:
klopt, deze staat in pnet als onlyusefor test